conf/messages

```diff
@@ -103,6 +103,9 @@ card-fees.para3 = You cannot pay using a personal credit card.
 card-fees.para4 = Allow 3 working days for your payment to reach HMRC’s bank account.
 card-fees.para2.bank-account = bank account
 
+#Select a method of payment after a card payment failure
+card-fail-choice.error = Select how you want to pay
```

Comment:
can you make this align to the existing notation, we already have messages for the payment failed screen and this is that page
```payment-failed.error```
or similar, also put it next to those error messages. We don't want a messy messages file like in pay-frontend